README.md

```diff
@@ -32,21 +32,17 @@ you use the default version.
 
 Simply running a container with the image will launch a Perl 6 REPL:
 
-    $ docker run -it rakudo-star
+    $ docker run --rm -it rakudo-star
     > say 'Hello, Perl!'
     Hello, Perl!
 
 You can also provide perl6 command line switches to `docker run`:
 
-    $ docker run -it rakudo-star perl6 -e 'say "Hello!"'
+    $ docker run --rm rakudo-star perl6 -e 'say "Hello!"'
```

Comment:
ditto

Reply:
And ditto. My idea was that probably most of those who read this don't realize that they create a new container with every `docker run`.